recipes/gf-complete/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
   "1.03":
-    url: "https://github.com/hrantzsch/gf-complete/archive/v1.03.tar.gz"
-    sha256: "396664e0933b8409470e6e0efc2227f787e2909cb1fd0d094d016eb6b5b34b2d"
+    url: "https://github.com/ceph/gf-complete/archive/58eebabd204998364355eee293402929502bf1a9.tar.gz"
+    sha256: "ba9d90b5818b8356073fdc21704bdd8b70e80aceb51700374386393f39cb1b0e"
```

Comment:
why changed sha for 1.0.3? also, why didn't use the tag previously used?

Reply:
As discussed in PR #1692.

gf-complete doesn't publish real releases. I previously bundled a release manually, because I didn't know about the practice of using `cci.<date>`.

To fix this, we decided to keep the package with the version `1.03`, but point it to the (github auto-bundled) commit where the version number was first increased to `1.03`.
`cci.20170410` is the latest commit.